src/middlewared/middlewared/plugins/device_/vrrp_events.py

```diff
@@ -289,6 +303,7 @@ async def _start_stop_vrrp_threads(middleware):
         if VrrpObjs.event_thread is None or not VrrpObjs.event_thread.is_alive():
             VrrpObjs.event_thread = VrrpEventThread(middleware=middleware, timeout=timeout)
             VrrpObjs.event_thread.start()
+            await middleware.call('vrrpthread.set_non_crit_ifaces')
```

Comment:
Isn't it better to have non-critical interfaces already before the thread is started to avoid race conditions?